extension.js

```diff
@@ -91,7 +91,7 @@ class Extension {
            .map(w => ({id: w.toString(),
                        ref: w,
                        title: w.get_meta_window().get_wm_class()}))
-           .filter(w => !w.title.includes('Gnome-shell'));
+           .filter(w => w.title != null && !w.title.includes('Gnome-shell'));
```

Comment:
In the rest of the file we try to use strict comparison (`===` and `!==`).

Otherwise, this could be written even simplier:
```suggestion
           .filter(w => w.title && !w.title.includes('Gnome-shell'));
```